core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -322,16 +375,56 @@ object IgnoreExecs {
     "CreateDataSourceTableAsSelectCommand"
   private val SetCatalogAndNamespace = "SetCatalogAndNamespace"
   private val ExecuteSetCommand = "Execute SetCommand"
-
-
-  val True = "true"
-  val False = "false"
-
-  def getAllIgnoreExecs: Set[String] = Set(AdaptiveSparkPlan, CollectLimit, ScanExistingRDD,
-    ExecuteCreateViewCommand, ExistingRDD, LocalTableScan, ExecuteCreateTableCommand,
+  private val ResultQueryStage = "ResultQueryStage"
+  private val ExecAddJarsCommand = "Execute AddJarsCommand"
+  private val ExecInsertIntoHadoopFSRelationCommand = "Execute InsertIntoHadoopFsRelationCommand"
+  private val ScanJDBCRelation = "Scan JDBCRelation"
+  private val ScanOneRowRelation = "Scan OneRowRelation"
+  private val CommandResult = "CommandResult"
+  private val ExecuteAlterTableRecoverPartitionsCommand =
+    "Execute AlterTableRecoverPartitionsCommand"
+  private val ExecuteCreateFunctionCommand = "Execute CreateFunctionCommand"
+  private val CreateHiveTableAsSelectCommand = "Execute CreateFunctionCommand"
+  private val ExecuteDeleteCommand = "Execute DeleteCommand"
+  private val ExecuteDescribeTableCommand = "Execute DescribeTableCommand"
+  private val ExecuteRefreshTable = "Execute RefreshTable"
+  private val ExecuteRepairTableCommand = "Execute RepairTableCommand"
+  private val ExecuteShowPartitionsCommand = "Execute ShowPartitionsCommand"
+  // DeltaLakeOperations
+  private val ExecUpdateCommandEdge = "Execute UpdateCommandEdge"
+  private val ExecDeleteCommandEdge = "Execute DeleteCommandEdge"
+  private val ExecDescribeDeltaHistoryCommand = "Execute DescribeDeltaHistoryCommand"
+  private val ExecShowPartitionsDeltaCommand = "Execute ShowPartitionsDeltaCommand"
+
+  def getAllIgnoreExecs: Set[String] = Set(AdaptiveSparkPlan, CollectLimit,
+    ExecuteCreateViewCommand, LocalTableScan, ExecuteCreateTableCommand,
     ExecuteDropTableCommand, ExecuteCreateDatabaseCommand, ExecuteDropDatabaseCommand,
     ExecuteCreateTableAsSelectCommand, ExecuteCreateDataSourceTableAsSelectCommand,
-    SetCatalogAndNamespace, ExecuteSetCommand)
+    SetCatalogAndNamespace, ExecuteSetCommand,
+    ResultQueryStage,
+    ExecAddJarsCommand,
+    ExecAddJarsCommand,
```

Comment:
Nit: Duplicate entry(this is a set so would be ignore later anyway :) )

Reply:
Thanks @nartal1 ! :)